### DISCUSSION TITLE
Convert RGB color values to HSL

### DISCUSSION THREAD

**Discussion by r-nl** [tailwindlabs/tailwindcss#10723] [Ideas]
Given that HSL/HSLA support is now global, it might be worth converting the internally-stored color values of tailwind into HSL from RGB, as advocated in Refactoring UI. This would make the hover values of colored tailwind classes far more human readable, and make quick tweaks quicker.

**Comment by FNDEVVE** [comment]
Agree. I am reading Refactoring UI now and I would love to see every Tailwind property with HSL.
